ATL/schema.sql

```diff
@@ -0,0 +1,52 @@
+create table Passengers
+(PassengersId int identity primary key,
+[Name] varchar(20) not null,
+[LastName] varchar(30) not null,
+[Age] int not null,
```

Comment:
Storing age like this is a really bad idea. Try to think about why that is, we'll discuss it during the class.